lib/dependabot/file_updaters/java/maven.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "dependabot/file_updaters/base"
+
+module Dependabot
+  module FileUpdaters
+    module Java
+      class Maven < Dependabot::FileUpdaters::Base
+        def self.updated_files_regex
+          [/^pom\.xml$/]
+        end
+
+        def updated_dependency_files
+          [updated_file(file: pom, content: updated_pom_content)]
+        end
+
+        private
+
+        def check_required_files
+          %w(pom.xml).each do |filename|
+            raise "No #{filename}!" unless get_original_file(filename)
+          end
+        end
+
+        def updated_pom_content
+          # TODO: Update the pom.xml file for the new version requirement
```

Comment:
This will be trivial - since Maven has a single requirements file all the work will be done in the `UpdateChecker`. This class is just going to do a substring replace.